### PR TITLE
#184 - Documentation - Complete

### PR DESCRIPTION
**Pull request by soris-codes**
@wimo7083 @aaron-suarez 
Created an OpenAPI 3.0 yaml file using Swagger. It still needs work.

### CODE REVIEW

**Comment by aaron-suarez** [comment]
I went ahead and created a new repo for the docs. We should keep the openapi.yaml file here with the code, but also update it in the docs repo. https://github.com/OperationCode/resource_api_docs

Why don't you go ahead and clone the docs repo and run it locally. Once you see what it's going to look like, you might have some edits you'll want to make that are pretty obvious. If you need help actually making those edits, feel free to let me know and we can jump on a screenshare together!

File: openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+              schema:
+                $ref: '#/components/schemas/Error'
+      security:
+      - ApiKeyAuth: []
+  /search:
+    get:
+      description: Searches the API for a given keyword
+      responses:
+        200: {}
+  /apikey:
```
Comment (aaron-suarez) [inline openapi.yaml]: We definitely need an example of what a proper request looks like for this endpoint

**Comment by soris-codes** [comment]
Hello Aaron!

If you have time today (Sunday) or tomorrow, I'd be happy to do a
screenshare to work on the docs. Going to clone the docs repo to my machine
and check it out now.

Thanks so much for getting it all started!
~Soris

**Comment by soris-codes** [comment]
Added updated openapi.yaml file from #1 on resource_api_docs repo to this PR.

**Pull request merged**